### ISSUE TITLE
Downloading generated maps on statistics

### ISSUE THREAD

**Issue by Iheanacho2027** [hzi-braunschweig/SORMAS-Project#1486]
Users should be able to download maps generated on the statistics menu. Also the maps generated on the statistics menu for states and LGAs do not clearly visualize names by using an external view, just as it is displayed in geographical systems like QGIS

**Comment by foyiri** [comment]
Yes Acho we had this discussion we want to be able to view state LGA and even ward that will really help with viewing where the cases are located on the map

**Comment by MartinWahnschaffeSymeda** [comment]
@Iheanacho2027 Adding a download option for the shown map makes a lot of sense.

What do you mean by "visualize names by using an external view"? Maybe you can provide an example?

**Comment by Iheanacho2027** [comment]
What i mean is names of locations e.g LGAs if considering the map of state should be shown inside the map like we have in QGIS. I think having a statistical map which is ready for use by users is long overdue and should be given priority

**Comment by MateStrysewskeSym** [comment]
@Iheanacho2027 Could you provide a screenshot or a quick mockup that demonstrates this?
